dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -2632,6 +2632,29 @@ paths:
             $ref: "#/definitions/Entry"
       security:
       - BEARER: []
+  /entries/{id}/categories:
```

Comment:
you can omit swagger.yaml for most new endpoints (forces us to use openapi elsewhere)

Reply:
Yeah, I missed this one, thanks.